openpype/hosts/houdini/api/plugin.py

```diff
@@ -187,13 +187,14 @@ def create(self, subset_name, instance_data, pre_create_data):
             self.customize_node_look(instance_node)
 
             instance_data["instance_node"] = instance_node.path()
+            instance_data["instance_id"] = instance_node.path()
```

Comment:
```suggestion
            instance_data["instance_id"] = instance_node.path()
            instance_data["family"] = self.family
```

I'd even argue that the Creator should also always be capable of defining `families` to like in the Maya USD PR I linked where the creator could provide families `["usd", "mayaUsd"]`  or alike to create a product type **usd**, also pass through any USD validators, but have a dedicated **mayaUsd** extractor for that Creator.

By allowing the Creator to define these instead of a post collector adding in `usd` for example also allows Attribute Definitions on plugin on the different families to show for that Creator because before publishing it then knows it has those families.

This could e.g. then also allow a Render Creator itself, to aside of just `render` family add `render.{renderer}` family like `render.arnold` to on the publish tab e.g. show Arnold specific attributes only if the render instance is set to render with arnold ❤️ 

However as far as I know (@iLLiCiTiT can probably confirm) the new publisher isn't capable yet of producing different attribute definitions per instance but does so PER creator currently (even though it does call the `get_attribute_defs()` method per instance - I recall hearing before that's still a bug to be fixed.